packages/components/src/border-control/styles.ts

```diff
@@ -29,41 +29,15 @@ const focusBoxShadow = css`
 	box-shadow: inset 0 0 0 ${ CONFIG.borderWidth } ${ COLORS.ui.borderFocus };
 `;
 
-export const borderControl = css`
-	position: relative;
-`;
+export const borderControl = css``;
```

Comment:
🧹 ?

Reply:
Sorry, I should have added an explanatory comment to this PR.

I couldn't see or recall why this style was in place and in my testing it didn't appear to have an impact now. So I've cleaned it up.

Reply:
Yes, sorry that was unclear! 🙈

Reply:
Thanks for the clarification.

The `borderControl` variable is required now after https://github.com/WordPress/gutenberg/pull/42348 adds a few styles there to wrangle the introduced `fieldset`. So it will live to see another day after I rebase this PR. 🙂